gradoop-checkstyle/src/main/resources/gradoop/checkstyle-suppressions.xml

```diff
@@ -11,4 +11,7 @@
   <suppress checks="IllegalCatch"
             files="PageRank.java"
             lines="50-70"/>
+  <suppress checks="IllegalCatch"
```

Comment:
Since we have this problem with every single gelly algorithm, the try/catch/throw RuntimeException pattern should be lifted into the gelly algorithm abstract class. This should reduce the number of suppressed checks.